lib/adapters/class-wp-com.php

```diff
@@ -116,6 +116,9 @@ public function query( $es_args ) {
 			$es_args = apply_filters( 'es_admin_wp_com_es_args', $es_args );
 
 			$response = es_api_search_index( $es_args, 'es-admin' );
+			if ( is_wp_error( $response ) ) {
+				return $es_args['name'];
```

Comment:
I think this should probably be: 

```suggestion
				return array();
```

Do you also want any logging to happen here?